lib/web-test/src/benchcontainer.rs

```diff
@@ -0,0 +1,53 @@
+use crate::prelude::*;
+use crate::system::web::create_element;
+use crate::system::web::dyn_into;
+use crate::system::web::NodeInserter;
+use crate::system::web::StyleSetter;
+use web_sys::HtmlElement;
+use super::Container;
+
+/// Html benchmark container.
```

Comment:
"Html benchmarks" sounds like there were html program we want to measure with benchmarks :) Maybe "Html container displaying benchmark results"?

Reply:
Fixed!